charts/zenko-nfs/values.yaml

```diff
@@ -5,6 +5,10 @@ replicaCount: 1
 
 service:
   type: NodePort
+  extra:
+    rquotad: 875
```

Comment:
Should these ports be configurable?

Reply:
Not particularly. Only in case of unexpected port conflicts

Reply:
Then they shouldn't be configurable.

Reply:
they should be configurable

Reply:
There is no need to make them configurable, and it helps to avoid possible configuration errors. The only scenario in which there could be port conflicts is at the pod level (containers within a pod with the same port), but that will be a conflict caused by us.

Besides that, this is the way we have being doing it in Zenko charts. PR's in the past have been requested changes like this one.